apps/forms-e2e/src/data-hunter/utils/helpers.ts

```diff
@@ -5,7 +5,6 @@ export const generateAccessibilityBuilder = async (page: Page) =>
   new AxeBuilder({ page }).withTags([
     'wcag2a',
     'wcag2aa',
-    'wcag2aaa',
```

Comment:
WCAG AAA er heldigvis ikke lovpålagt 😊

Reply:
Tenkte det er likevel nyttig å ha det med for å hindre at de blir innført heller enn at de må fikses i ettertid. Det er fordi noen av feilene som SiteImprove (Noen lignende Monsido som vi brukte tidligere) hadde fanget opp og vi fikset var også AAA. Men vi kan også spørre teamet for å være sikker.